sirepo_bluesky/utils/json_yaml_converter.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import json
+import os
+from enum import Enum
+
+import yaml
+
+
+class Filetype(Enum):
+    JSON = 1
+    YAML = 2
+
+
+def json_to_yaml(json_fp, yaml_fp, **kwargs):
+    """
+    Converts a .json file into a .yaml or .yml file.
+
+    Parameters
+    ----------
+    json_fp : str
+            Filepath of .json file.
+    yaml_fp : str
+            Filepath of new .yaml or .yml file.
```

Comment:
The descriptions here and in the functions below seem to be over-indented. Can we use 4 blank spaces for that?